Mathlib/Topology/UnitInterval.lean

```diff
@@ -172,6 +212,25 @@ theorem mul_pos_mem_iff {a t : ℝ} (ha : 0 < a) : a * t ∈ I ↔ t ∈ Set.Icc
 theorem two_mul_sub_one_mem_iff {t : ℝ} : 2 * t - 1 ∈ I ↔ t ∈ Set.Icc (1 / 2 : ℝ) 1 := by
   constructor <;> rintro ⟨h₁, h₂⟩ <;> constructor <;> linarith
 
+instance : LinearOrderedCommMonoidWithZero I where
```

Comment:
Please reuse the existing `LinearOrder` instance. Something like
```
  __ := inferInstanceAs (LinearOrder I)
```
should work. Otherwise we risk introducing two incompatible `LinearOrder I` structures.

Reply:
Thanks, that worked!